missioncontrol_client/__init__.py

```diff
@@ -13,6 +86,17 @@ def __init__(self, mc_base, jwt=None):
         self.s = requests.session()
         self.jwt = None
 
+    @classmethod
+    def from_environ(cls, ignore_ssl=False):
```

Comment:
Somehow I thought this was already merged somewhere?

Reply:
There's a separate function to do this. I think a classmethod constructor is a lot more obvious and discoverable.

Reply:
Perhaps add a deprecated comment to the existing function then?

Reply:
can do!